src/test/java/paqua/loan/amortization/LoanAmortizationCalculatorTest.java

```diff
@@ -347,4 +347,19 @@ void shouldFailWhenLoanValuesAreNull() {
         Assertions.assertThrows(LoanAmortizationCalculatorException.class, () ->
             calculator.calculate(new Loan(null, null, null, null, null)));
     }
+
+
+	@Test
+	public void shouldTakeDoubleValuesForLoan(){
+		Loan loan = Loan.builder()
+				.amount(500000.32)
+				.rate(4.56)
+				.term(10)
+				.build();
+		LoanAmortization amortization = calculator.calculate(loan);
+		assertNotNull(amortization);
+
```

Comment:
Looks good. But I think we can do better.
I see two possible variants:
1. Add an assertion. Here we can use an existing test case from one of the tests with an equals assertion of the calculated and expected result. 
Therefore we can verify that everything is correct with double values.
2. Or move this test to the LoanTest.class and verify only the correctness of set values without actual calculation. 
In this variant, it would be more like a unit test instead of an integration test.